src/components/Cards.js

```diff
@@ -0,0 +1,47 @@
+import "./Cards.css";
+import { getDate, getElapsedTime } from "../utils";
+import { useState } from "react";
+
+function Card({ data }) {
+  const [hover, setHover] = useState(false);
+  const imageStyle = {
+    backgroundColor: "lightgray 50%",
+    backgroundImage: `url(${data.imageSource || "images/card-default.png"})`,
+    backgroundPosition: "center",
+    backgroundSize: "cover",
+    backgroundRepeat: "no-repeat",
+  };
+  return (
+    <div
+      className="card"
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+    >
+      <div className="card-image-area">
+        <img
+          className={"card-image " + (hover ? "hover-image" : "")}
+          src={data.imageSource || "images/card-default.png"}
+          alt="card"
+        ></img>
```

Comment:
self closing tag 를 사용해보시면 좋습니다.
```suggestion
        <img
          className={"card-image " + (hover ? "hover-image" : "")}
          src={data.imageSource || "images/card-default.png"}
          alt="card"
        />
```